x/examples/ws2endpoint/main.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"github.com/Jigsaw-Code/outline-sdk/x/config"
+	"golang.org/x/net/websocket"
+)
+
+func main() {
+	addrFlag := flag.String("localAddr", "localhost:8080", "Local proxy address")
+	transportFlag := flag.String("transport", "", "Transport config")
+	endpointFlag := flag.String("endpoint", "", "Address of the target endpoint")
+	pathPrefix := flag.String("path", "/", "Path where to run the Websocket forwarder.")
```

Comment:
*nit* Remove the dot to align with other flag description, or you may append the dot to other flags too.

```suggestion
	pathPrefix := flag.String("path", "/", "Path where to run the Websocket forwarder")
```